csharp/src/Microsoft.ML.OnnxRuntime/OrtEnv.shared.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.OnnxRuntime
+{
+    /// <summary>
+    /// This class handles the process-global ONNX Runtime environment instance (OrtEnv)
+    /// </summary>
+    public sealed class OrtEnv : SafeHandle
```

Comment:
>public sealed class OrtEnv : SafeHandle [](http://example.com/codeflow?start=4&length=39)

OrtEnv class already exists in the C# API. You may want to add new functionality there.

Reply:
Yes, I moved the class from being in this file to its own class file (I thought this conformed more to the rest of the project). I now see that the file `OnnxRuntime.shared.cs` does not contain a lot after i migrated `OrtEnv` to its own file. Should i move the code back from `OrtEnv.shared.cs` to `OnnxRuntime.shared.cs` and rename it?

- [ ] Rename `OnnxRuntime.shared.cs`?
- [ ] Remove `OrtEnv.shared.cs` ?

Reply:
It makes it harder to review your specific change. I do not have an objection at this time on moving in a separate PR.